### PR TITLE
Update friendsofsymfony/http-cache to v2.0@beta

### PR DESCRIPTION
**Pull request by yukoff**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | #2106, #3070
| License | MIT
| Documentation PR | -

#### What's in this PR?

Update friendsofsymfony/http-cache to v2.0@beta

#### BC Breaks/Deprecations

None

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```
Comment (wachterjohannes) [inline composer.json]: is this really needed?

Comment (yukoff) [inline composer.json]: I wouldn't say it is required, but http-cache v2.0 will set this constraint anyway. On the other hand, php 5.5 is EOL

Comment (wachterjohannes) [inline composer.json]: @yukoff your right but we dont want to change php deps inside a minor release. we currently talking about ban 5.5 for 2.0 release.

It should be possible to use both http-cache version 2 and version 1? isnt it?

Comment (yukoff) [inline composer.json]: Ah, I got your point. Unfortunately, 2.0 version of fos/http-cache brings few BC breaks :( For example, it dropped `...Interface` suffix on all interface classes and a few other changes.

Comment (wachterjohannes) [inline composer.json]: hmm thats weird (: lets include also @danrot and @chirimoya in this discussion. maybe we can include this not before 2.0.

Comment (danrot) [inline composer.json]: Sounds to me like this is not possible before 2.0... If @chirimoya also agrees we can tag this PR on the 2.0 milestone.

**Comment by danrot** [comment]
Hey @yukoff, @trickreich has used your PR as a starting point for a bigger cache refactoring. That's PR number #3428, you should see your commits there. Therefore I can close this PR. Thanks for the contribution!

**Comment by yukoff** [comment]
Hi @danrot Thanks for the update, looking forward fot that! BTW I think one more commit should be considered then #3230 (207fb4ae8ce3ee9185d6920a99d8291be450e3ce)  to get rid of guzzle3 leftovers.

**Pull request closed without merging**